production/helm/loki/templates/tokengen/clusterrole-tokengen.yaml

```diff
@@ -1,7 +1,7 @@
-{{ if and (and .Values.enterprise.tokengen.enabled .Values.enterprise.enabled) (not .Values.rbac.namespaced)}}
+{{ if and .Values.enterprise.tokengen.enabled .Values.enterprise.enabled }}
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
+kind: {{ if not .Values.rbac.namespaced }}Cluster{{ end }}Role
```

Comment:
did you manage to successfully build the template with `namespace=true` and false?